src/ChannelServer/World/SpawnZone.cs

```diff
@@ -0,0 +1,219 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Melia.Shared.Data.Database;
+using Melia.Shared.Util;
+using Melia.Shared.World;
+
+namespace Melia.Channel.World
+{
+
+
+	public struct SpawnData
+	{
+		public string spawnName;
+		public string mapName;
+		public string monsterName;
+		public int count;
+		public int countVariation;
```

Comment:
You should always document code which's behavior isn't immediately obvious. Fields like this or methods like `ProcessMonsters` could be anything, and without documentation you need to read the code to learn what they're even doing.